viam-cartographer/src/io/submap_painter.h

```diff
@@ -1,33 +0,0 @@
-// This is an experimental integration of cartographer into RDK.
-#ifndef VIAM_CARTOGRAPHER_IO_SUBMAP_PAINTER_H_
```

Comment:
Can you please provide the diff of this file & the file it was overriding in cartographer?